### PR TITLE
Learners Learning Path

### PR DESCRIPTION
**Pull request by FeLungs**
Algorithm + Visualization for upcoming demo

### CODE REVIEW

**Comment by FeLungs** [comment]
Need to handle when the actor is a Group.

**Comment by FeLungs** [comment]
still need to add the updates to change log. Want to make sure CI passes before this draft PR is confirmed

**Comment by FeLungs** [comment]
Should be in a good state for connecting Learning Path algorithm to appropriate viz

File: src/com/yetanalytics/dave/func.cljc [hunk 1]
```diff
@@ -568,6 +570,227 @@
...
+    (let [{:keys [timestamp id actor verb object]} (common/parse-statement-simple statement)
+          {:keys [verb-id verb-name]}               verb
+          relevant-actor-info                       (common/handle-actor actor)
+          many-actor-agents?                        (-> relevant-actor-info first vector?)
+          [obj-name id-or-members]                  (common/handle-object object)
+          obj-member-names                          (when (vector? id-or-members)
+                                                      (apply str (interpose "," (map first id-or-members))))
+          object-display                            (or obj-member-names obj-name)]
+      (if many-actor-agents?
+        (do
```
Comment (milt) [inline src/com/yetanalytics/dave/func.cljc]: @FeLungs this conditional branch doesn't do anything as far as I can tell. It just `doseq`s over something and then returns `this`, unmodified. Remember that clojure `defrecord`s (as opposed to `deftypes`) are immutable and can be thought of inside methods as a clojure map.

**Pull request merged**